src/draw/lv_draw_arc.c

```diff
@@ -67,6 +67,7 @@ LV_ATTRIBUTE_FAST_MEM void lv_draw_arc_dsc_init(lv_draw_arc_dsc_t * dsc)
     dsc->color = lv_color_black();
 }
 
+#if LV_USE_GPU_SDL_RENDER == 0
```

Comment:
if we will add support for other GPUs too it will be cumbersome to update all these `#if`s. E.g.
```c
#if LV_USE_GPU_SDL_RENDER == 0 && \
 LV_USE_GPU_OPENGLES_RENDER == 0 && \
 LV_USE_GPU_WEBGL_RENDER == 0
```

What about using a define like `LV_USE_EXTERNAL_RENDER` here and `LV_USE_GPU_SDL` to enable SDL drawers?

Reply:
@kisvegabor I would highly agree on that. I'll change to this definition. 
P.S. Please tell me more about your thoughts by reviewing so I can change such thing to have consistent styles with main repo :)

Reply:
Thank you. I can proceed a little it slow with the review because I'm not so experienced with SDL. 
But I'm on it :slightly_smiling_face: